ci-operator/step-registry/baremetalds/devscripts/setup/baremetalds-devscripts-setup-commands.sh

```diff
@@ -54,14 +54,23 @@ timeout -s 9 175m ssh $SSHOPTS root@$IP bash - << EOF |& sed -e 's/.*auths.*/***
 
 set -ex
 
-yum install -y git
+yum install -y git sysstat sos
+systemctl start sysstat
 
 mkdir -p /tmp/artifacts
 
 mkdir dev-scripts
 tar -xzvf dev-scripts.tar.gz -C /root/dev-scripts
 chown -R root:root dev-scripts
 
+NVME_DEVICE="/dev/nvme0n1"
```

Comment:
This is the portion I was referring to (the one to be moved into baremetalds-packet-setup Ansible playbook)

Reply:
I think that makes sense.

Can we merge this version (after rehearse finishes) and I promise to refactor it this afternoon and have a PR ready for you to look @ on Monday? 

I'd like to see the impacts on the 4.4 and 4.5 periodics over the weekend

Reply:
Ok, let's see the rehearse then